examples/plot_firing_pattern.py

```diff
@@ -15,13 +15,12 @@
 # Let us import hnn_core
 
 import hnn_core
-from hnn_core import read_params, Network
-from hnn_core.network_builder import NetworkBuilder
+from hnn_core import read_params, Network, simulate_dipole
 
 hnn_core_root = op.dirname(hnn_core.__file__)
 
 ###############################################################################
-# Then we read the parameters file
+# Then we read the parameters file and enable somatic voltage recording
```

Comment:
shouldn't this go to the blurb below now that we do it through `simulate_dipole`?